client/my-sites/domains/domain-management/list/style.scss

```diff
@@ -457,6 +457,19 @@
 }
 .list__status-cell {
 	flex: 40 40 0;
+	&__bubble {
+		display: flex;
+		width: 20px;
+		background: #d67709;
```

Comment:
We can use `var( --studio-orange-40 )` here

Reply:
You're right!